Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/clientmetadata/ClientVersionWrapper.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.datadonation.analytics.modules.clientmetadata
+
+import android.os.Build
+import dagger.Reusable
+import de.rki.coronawarnapp.BuildConfig
+import javax.inject.Inject
+
+@Reusable
+class ClientVersionWrapper @Inject constructor() {
```

Comment:
Do we need an extra class for this?

We also already have `ApiLevel`.

Reply:
ah nice, didn't know about ApiLevel 👍